### PR TITLE
Revive code_* modules. Add a test for codebuild and push to ECR

### PR DESCRIPTION
**Pull request by mtp1376**

### CODE REVIEW

File: src/modules/iasql_functions/iasql.ts [hunk 1]
```diff
@@ -77,9 +77,7 @@ export async function modules(all: boolean, installed: boolean, dbId: string) {
     }));
   if (all) {
     // TODO: Remove this filter once two-way mode is standard
```
Comment (depombo) [inline src/modules/iasql_functions/iasql.ts:79]: remove the TODO now that you did it :)

File: test/modules/aws-codebuild-integration.ts [hunk 2]
```diff
@@ -361,13 +361,87 @@ phases:
   );
 
   it('starts a transaction', begin());
+  it(
+    'creates a project that pushes to ecr',
+    query(
```
Comment (depombo) [inline test/modules/aws-codebuild-integration.ts:366]: you should probably just use the `generate_put_ecr_image_build_spec` function here?

Comment (dfellis) [reply test/modules/aws-codebuild-integration.ts:366]: If that is used elsewhere in the test suite, then I would say it's fine. Better to exercise more possible paths.

Comment (mtp1376) [reply test/modules/aws-codebuild-integration.ts:366]: It's "building" a specific folder. In this case I've just pulled an image, retagged and pushed (to reduce the rate of errors). For that case we'll need to add Github credentials which will test multiple things.
If that's not a blocker, I'd suggest we go with the current version.

Comment (depombo) [reply test/modules/aws-codebuild-integration.ts:366]: that makes sense 👍

**Review by depombo — APPROVED** [review]

**Review by aguillenv — APPROVED** [review]

**Pull request merged**